cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/oci_repo.go

```diff
@@ -261,7 +261,14 @@ func (r *OCIRegistry) downloadChart(chart *repo.ChartVersion) (*bytes.Buffer, er
 
 	// trim the oci scheme prefix if needed
 	getThis := strings.TrimPrefix(u.String(), fmt.Sprintf("%s://", registry.OCIScheme))
-	return r.helmGetter.Get(getThis, clientOpts...)
+	log.Infof("about to call helmGetter.Get(%s)", getThis)
```

Comment:
Not sure if this log should be a debug-level one ?

Reply:
all these pesky log messages will be gone by end of next week. They're here only while I was actively working on this feature. See my 2nd action item in https://github.com/vmware-tanzu/kubeapps/issues/5007#issuecomment-1186031237. I just haven't got to it yet